Makefile

```diff
@@ -57,10 +58,10 @@ ifeq ($(USE_DOCKER), 1)
 endif
 
 ifeq ($(NO_DOCKER), 1)
-  DOCKER_CMD =
+  DOCKER_CMD = CGO_ENABLED=0 && 
```

Comment:
This doesn't appear to actually do anything, we already have cgo disabled by line 39 don't we?

What's the `&&` for in this case?

Reply:
that's a good point about line 39, i can probably remove this.

the `&&` just says "do the thing to the left, and if it fails return an error before doing the thing on the right", i was trying to make sure we had the env variable set for non-container builds. i kept getting an error that way it was expressed before.

Reply:
i ran a couple tests, and fwiw that variable set on L39 does not get propagated to the build environment. that said, i think we don't need it set explicitly, i tried doing a few test builds and they looked the same whether i had it enabled or not. i will remove the part on L61 just to reduce confusion here.

Reply:
i'm doing some further testing and i'm doubting my previous results. when i set up the build container with the proper env variables i get a clean build, but if i remove the `CGO_ENABLED` i get some warnings about the shared libraries. i am starting to think we need all the env variables loaded into `DOCKER_CMD` when `NO_DOCKER=1`. i will do some further testing though to confirm.